deliver/lib/deliver/upload_screenshots.rb

```diff
@@ -9,23 +32,55 @@ def upload(options, screenshots)
       v = app.edit_version
       UI.user_error!("Could not find a version to edit for app '#{app.name}'") unless v
 
-      UI.message("Starting with the upload of screenshots...")
+      @checksums_local = ScreenshotMD5s.new
+      @checksums_remote = ScreenshotMD5s.new
+
+      screenshots.each do |screenshot|
+        md5 = Spaceship::Utilities.md5digest(screenshot.path)
```

Comment:
Is the path unique here? e.g. relative path vs absolute path? Should we call `File.expand_path(screenshot.path)` to make sure it's always absolute, and therefore the string matches?